CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_marketo v0.9.1
+
+## Bug Fix
+[PR #26](https://github.com/fivetran/dbt_marketo/pull/26) includes the following non-breaking change:
+- Due to a "system glitch" within Marketo, the same `lead_id` can be deleted more than once. This may introduce multiple records for the same `lead_id` in the `int_marketo__lead` which can cause the downstream test `marketo__leads.lead_id.unique` to fail.  The bug fix adds a `distinct lead_id` to the `deleted_leads` CTE in the `int_marketo__lead` model (https://github.com/fivetran/dbt_marketo/issues/25). 
```

Comment:
Do you by chance have a link to Marketo documenting this system glitch? If so, this would be a great place to link it for additional details.

Reply:
Unfortunately, I don't have a link to any documentation regarding this glitch. It was communicated to our business partner in a case that was submitted to Marketo support.